### ISSUE TITLE
Missing results/messages/errors for SQL Server

### ISSUE THREAD

**Issue by Radioleao** [dbeaver/dbeaver#18890]
### Description

Using DBeaver on SQL Server, I have an error about missing messages and results.
Note that:
1. The error happens both with Execute SQL Statement and Execute SQL Script
2. The error happens both with or without separators.

When I run multiple queries, I get only the first result.
![image](https://user-images.githubusercontent.com/8766975/215043216-24e670ec-bbc8-40b6-ba03-6927b1516d5f.png)

When I run multiple queries and I got any error, I get only the first result and the error message is not shown.
![image](https://user-images.githubusercontent.com/8766975/215044104-6af89c8b-30c7-4d5b-8975-de93aab56909.png)

The error is not about missing execution of following statements.
I tested a temp table creation both before and after raiserror, and both tables are created.
The scripts are:
```
select 1 
select 1 as A into #t
raiserror('Test', 16, 1)

select 1 
raiserror('Test', 16, 1)
select 2 as B into #v
```
Table #t has been created as expected.
Table #v has been created as expected.

Further error:
I get a null pointer exception trying to test multiple select in a more complex script:
![image](https://user-images.githubusercontent.com/8766975/215064352-d2d1c403-0563-4bf8-81cb-96997a27e167.png)

Foot Notes:
1. The issue doesn't happen when DBeaver is just started. But after some event (I have not understood which one) the issue happens everytime until I reboot DBeaver;
2. The error has already been posted ( #18816 ) but his solution didn't work for me.
3. SQL Server version is 2019

Thanks

### DBeaver Version

Community edition 22.3.3

### Operating System

Windows 10

### Database and driver

Database: MS SQL 2019
Driver: Microsoft JDBC Driver for SQL Server (MSSQL)

### Steps to reproduce

1. Connect to SQL Server
2. Run a Statement with 2 select queries and / or a raiserror. Examples:
```
select 1 as A;
select 2 as A;

select 1 as A;
raiserror('test error', 16, 1);
```
3. Check that only first select result is shown.

### Additional context

Note that the error does't happens when the application is just started.
After first time, it happens everytime.

When I understand new information I will update this issue report.

**Comment by E1izabeth** [comment]
Thank you for bug report.

**Comment by Radioleao** [comment]
In this thread I added a description for a further issue about resultsets, occurred during some attemps to get more infos.

**Comment by serge-rider** [comment]
@Radioleao do you use our SQL Server driver or you created a custom driver with downloaded jar?
Could you send a screenshot of connection config script (you can delete all non-public info from there)?

Also could you send the error log (https://github.com/dbeaver/dbeaver/wiki/Log-files)

Thank you

**Comment by serge-rider** [comment]
The problem caused by SQ LServer driver which handles warnings (`raise`) as errors. This replaces last query results with error panel.
I fixed driver config + now query warnings are printed in the output log.

@Radioleao howver it would be great to have logs because of NullPointerError in log which also should be addressed and we need logs for that.

**Comment by sagitarriuss** [comment]
@serge-rider @Matvey16 @Radioleao
The issue is reproduced again in the 24.1.x release (last tested on 24.1.4). Even the 'print 1' statement doesn't show output. A new ticket is needed. Driver: MS SQL Server - com.microsoft.sqlserver.jdbc.SQLServerDriver (tested on v12.7.0 and v12.8.0)

**Comment by sagitarriuss** [comment]
Update: The print statement output issue has been resolved in DBeaver CE 2.1.5.
